src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27&#34; Retina 5K Monoblock (MNED2UA/A)"
+      >
+      <p class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="product-code">Product code: 195434</p>
+
+      <div class="stars stars--4">
+        <div class="stars-star"></div>
+        <div class="stars-star"></div>
+        <div class="stars-star"></div>
+        <div class="stars-star"></div>
+        <div class="stars-star"></div>
+
+        <p class="stars__reviews">Reviews: 5</p>
+      </div>
+
+      <div class="price">
```

Comment:
<img width="697" alt="image" src="https://user-images.githubusercontent.com/102411130/234195742-c878c38f-b8ae-4c1f-88e4-a1945c945950.png">
no need to create redundant bem blocks